build.rs

```diff
@@ -0,0 +1,26 @@
+use std::io;
+use winres::WindowsResource;
+
+// Issues with winres to keep in mind:
+// https://github.com/mxre/winres/issues/32
+// https://github.com/mxre/winres/issues/40
+
+fn main() -> io::Result<()> {
+    if std::env::var("CARGO_CFG_TARGET_FAMILY").unwrap() == "windows" {
+        let mut res = WindowsResource::new();
+        let env = std::env::var("CARGO_CFG_TARGET_ENV").unwrap();
+        match env.as_str() {
+            "gnu" => {
+                // Not sure whether this works the same if build on windows; needs testing
+                // Perfectly fine if cross compiling from linux
```

Comment:
![explorer_2omhbhc2SJ](https://user-images.githubusercontent.com/47116127/200296911-dd728155-92bd-4f62-9f82-89f2d05fb936.png)

Tested on Windows, it works!

Reply:
Great!

Reply:
Let's leave this for now, you probably build it for `msvc` since that's the default on Windows.

Reply:
You are right, my bad, I was using the wrong target
This fails on my end

![Screenshot 2022-11-07 162020](https://user-images.githubusercontent.com/47116127/200309232-b5b718b5-fd55-4d88-867a-338316166780.png)

Reply:
> You are right, my bad, I was using the wrong target This fails on my end

I'm confused, can I know which target you used here?